bin/cml/runner.js

```diff
@@ -443,7 +443,7 @@ exports.builder = (yargs) =>
       },
       driver: {
         type: 'string',
-        choices: ['github', 'gitlab'],
+        choices: ['github', 'gitlab', 'bitbucket'],
```

Comment:
doesn't this depend on #798?

Reply:
It could go independently throwing a non supported exception. But I can postpone it to be added in #798